tests/airgapped/1.9/pipelines/pipelines-runner/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:3.11.9-alpine
+RUN pip install kfp==2.8.0
```

Comment:
I think the newer version of pipelines is installing `2.7.0` as you can see in the error logs in [your comment](https://github.com/canonical/bundle-kubeflow/issues/915#issuecomment-2310554665)